src/tree/SlotTreeItemBase.ts

```diff
@@ -148,6 +148,15 @@ export abstract class SlotTreeItemBase extends AzExtParentTreeItem implements IP
         await client.updateApplicationSettings(settings);
     }
 
+    public async getApplicationLanguage(context: IActionContext): Promise<string> {
```

Comment:
We try not to bloat the tree item classes, so please move this out of this file closer to where it's actually used.

Reply:
Ok, I made this changed and moved the getApplicationLanguage() function to the downloadProject.ts file (formerly known as cloneLocally.ts)